toolchain/personae.md

```diff
@@ -0,0 +1,56 @@
+---
+title: The Good Docs Toolchain
+---
+
+This document summarizes the personae for The Good Docs Toolchain, a collection of tools designed to help users create better technical documentation.
+
+The following are the primary types of users (i.e. personae) targeted by this toolchain. Each has a number of distinct needs, as outlined in the below sections.
+
+## Tech writer
+
+The **Tech Writer** is an individual whose primary role is authoring technical documentation. These may be volunteers in open source projects or employees in commercial organizations. They will typically have a certain level of technical proficiency, although this will range from a simple understanding of technical concepts to full proficiency with tools such as the command line.
+
+As a tech writer...
+
+- I want to use a simple tool to manage docs-as-code so I can concentrate on writing.
+- I want to have access to spell/grammar check so I can take advantage of existing tools.
+- I want to have the option to use the editor of my choice so I don't have to learn something new.
+
+## Developer
+
+Often times **Developers** are called on to document the code or application they've created. In business this may be a matter of convenience, where in open source projects there may simply not be anyone else available. But in either case this is a secondary task added to their main roles, often without enough thought given to the effort required to produce something of quality. These individuals are (obviously) very technically savvy, although this doesn't preclude them from wanting to use GUI tools.
```

Comment:
```suggestion
**Developers** are often called on to document the code or application they've created. In business this may be a matter of convenience, where in open source projects there may not be anyone else available. But in either case this is a secondary task added to their main roles, often without enough thought given to the effort required to produce something of quality. These individuals are very technically knowledgeable, although this doesn't preclude them from wanting to use GUI tools, or other assistance to write well.
```

Reply:
I have a dissenting opinion in the world of engineering that all engineers should always be documenting...  and that it isn't a nice to have or optional. But it is something that should be part of what engineers do.  That is my point-of-view and maybe not the good-docs project?

Reply:
+1 to @mgan59 . It takes a village to raise good docs. Engineers, should have a significant role to play if quality docs are to be created efficiently.